scanners/subfinder/parser/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
if the parser doesn't need external dependencies I'd also not use a package.json
Not required and it's just additional bloat